tests/test_yapapi.py

```diff
@@ -0,0 +1,11 @@
+import toml
+from pathlib import Path
+
+from yapapi import __version__
+
+
+def test_version():
+    with open(Path(__file__).parents[1] / "pyproject.toml") as f:
```

Comment:
Maybe it would be better to look for "pyproject.toml" relative to "yapapi/\_\_init\_\_.py", not this test file (in case it is moved):
```
    with open(Path(yapapi.__file__).parents[1] / "pyproject.toml") as f:
```

Reply:
well, that's a very slight difference but sure...

Reply:
(updated)